code/__DEFINES/wounds.dm

```diff
@@ -54,50 +71,213 @@
 #define BIO_BONE (1<<0)
 /// Has flesh - allows the victim to suffer fleshy slash pierce and burn wounds
 #define BIO_FLESH (1<<1)
-/// Self explanatory
-#define BIO_FLESH_BONE (BIO_BONE | BIO_FLESH)
 /// Has metal - allows the victim to suffer robotic blunt and burn wounds
 #define BIO_METAL (1<<2)
 /// Is wired internally - allows the victim to suffer electrical wounds (robotic T1-T3 slash/pierce)
 #define BIO_WIRED (1<<3)
-/// Robotic: shit like cyborg limbs, mostly
-#define BIO_ROBOTIC (BIO_METAL|BIO_WIRED)
 /// Has bloodflow - can suffer bleeding wounds and can bleed
 #define BIO_BLOODED (1<<4)
 /// Is connected by a joint - can suffer T1 bone blunt wounds (dislocation)
 #define BIO_JOINTED (1<<5)
+/// Robotic - can suffer all metal/wired wounds, such as: UNIMPLEMENTED PLEASE UPDATE ONCE SYNTH WOUNDS 9/5/2023 ~Niko
+#define BIO_ROBOTIC (BIO_METAL|BIO_WIRED)
+/// Self explanatory
+#define BIO_FLESH_BONE (BIO_BONE|BIO_FLESH)
 /// Standard humanoid - can suffer all flesh wounds, such as: T1-3 slash/pierce/burn/blunt. Can also bleed
 #define BIO_STANDARD (BIO_FLESH_BONE|BIO_BLOODED)
+/// Standard humanoid limbs - can suffer all flesh wounds, such as: T1-3 slash/pierce/burn/blunt. Can also bleed, and be dislocated
+#define BIO_STANDARD_LIMB (BIO_STANDARD|BIO_JOINTED)
```

Comment:
```suggestion
/// Standard humanoid limbs - can suffer all flesh wounds, such as: T1-3 slash/pierce/burn/blunt. Can also bleed, and be dislocated. Human arms and legs
#define BIO_STANDARD_JOINTED (BIO_STANDARD|BIO_JOINTED)
```
The term limb is used in the code to refer to heads and chests throughout the code, should clarify that here